settings.gradle

```diff
@@ -0,0 +1,12 @@
+rootProject.name = "ktlint-gradle-samples"
+
+include(':kotlin-ks')
+project(':kotlin-ks').projectDir = new File('./samples/kotlin-ks')
```

Comment:
Can't you just use `project(':kotlin-ks').projectDir = file('./samples/kotlin-ks')`?

Reply:
Changed in 471d810